browser/brave_app_controller_mac.mm

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#import "brave/browser/brave_app_controller_mac.h"
+
+#include <string>
+
+#include "base/strings/sys_string_conversions.h"
+#import <Sparkle/Sparkle.h>
+
+@implementation BraveAppController
+
+- (void)applicationDidFinishLaunching:(NSNotification*)notify {
+  [super applicationDidFinishLaunching:notify];
+
+  [self startBraveUpdater];
+}
+- (void)applicationWillFinishLaunching:(NSNotification*)notification {
+  [super applicationWillFinishLaunching:notification];
+
+  [self initializeBraveUpdater];
+}
+
+- (void)initializeBraveUpdater {
+  constexpr int kBraveUpdateCheckInterval = 60;
+  NSURL* url = [NSURL URLWithString:@"https://example.com"];
```

Comment:
Can we get this changed to http://localhost:9090 for testing?

Reply:
Done. you can change feed url in the `src/brave/brave_init_settings.gni'